tools/topology/sof-byt-codec.m4

```diff
@@ -65,8 +65,8 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 # PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
```

Comment:
@juimonen Please also change the comments here.

Reply:
ok